### DISCUSSION TITLE
Initial portal release milestone

### DISCUSSION THREAD

**Discussion by clyne** [ProjectPythia/pythia-foundations#68] [General]
All, per our discussion at the IWG I took the liberty of creating a GitHub milestone, InitialPortalRelease, and gave my best shot at identifying the open issues and PRs that we want to address by the EC annual meeting on June 16. Feel free to comment here on what was and was not assigned to the June 16 milestone, and/or discuss at the hackathon next week. Perhaps a further step is to prioritize (high, medium, low) the identified milestones?

**Comment by brian-rose** [comment]
Looks great! Once we have the template finalized (which I think it is, just waiting on a final review), we should be able to move quickly through adapting all the open PRs and populating each chapter of the "Core Scientific Packages" section. I expect this will be a major thrust of next week's hackathon.

We *don't* yet have open issues or PRs for the outstanding chapters of the "Foundational Skills" section (Jupyter, GitHub). Here, we may be authoring from scratch and it's not clear how much we'll get done by June 16. But that's another potential focus for the hackathon.
